stock_picking_type_routing_operation/models/stock_location.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models, fields
+
+
+class StockLocation(models.Model):
+
+    _inherit = 'stock.location'
+
+    routing_operation_picking_type_id = fields.Many2one(
+        'stock.picking.type',
+        string='Routing operation',
+        help="Change destination of the move line according to the"
+             " default destination setup after reservation occurs",
+        # TODO add domain ?
```

Comment:
Yes according to https://github.com/guewen/stock-logistics-warehouse/pull/2/files#diff-b4c9de75e885ff06214334b8e7968993R26-R27